jhu/tests/test_smooth.py

```diff
@@ -1,16 +1,13 @@
 import pytest
 
-from os import listdir
 from os.path import join
 
 import numpy as np
 import pandas as pd
-from delphi_jhu.run import run_module
 
 class TestSmooth:
     def test_output_files_smoothed(self, run_as_module):
-
-        dates = [str(x) for x in range(20200303, 20200310)]
+        dates = [str(x) for x in range(20200304, 20200311)]
```

Comment:
What's this about?

Reply:
Ah, it was from [this issue](https://github.com/cmu-delphi/covidcast-indicators/issues/243). I wasn't sure which date was the latest, I'll switch it back.